test/integration/diag_nodes_test.go

```diff
@@ -16,6 +16,8 @@ import (
 )
 
 func TestDiagNodeConditions(t *testing.T) {
+	t.Skip("can't run containerized")
```

Comment:
do we run this non-containerized?

Reply:
> do we run this non-containerized?

CI attempts to.  There are four tests that attempt it.  Three now have defense baked in at runtime.  This is the only one we're losing.

Reply:
Followup issue to fix it?